os.kieserver.webapp/configure.sh

```diff
@@ -12,7 +12,7 @@ if [[ "$JBOSS_EAP_VERSION" == "6."* ]]; then
     BPMSUITE_BASE_DIR="jboss-eap-6.4"
     BPMSUITE_DEPLOYABLE_VERSION="eap6.x"
     else
-        BPMSUITE_BASE_DIR="jboss-eap-7.0"
+        BPMSUITE_BASE_DIR="jboss-eap-7.1"
```

Comment:
This script is intended to be used with kieserver 6.4 series, and the EAP7 version for kieserver64 was changed to 7.1?

Reply:
Ah, you're right. We might have to rework this based on jboss product too.